connect/mirror/src/main/java/org/apache/kafka/connect/mirror/ConfigPropertyFilter.java

```diff
@@ -25,8 +25,20 @@
 @InterfaceStability.Evolving
 public interface ConfigPropertyFilter extends Configurable, AutoCloseable {
 
+    /**
+     * Specifies whether to replicate the given topic configuration.
+     * This does not handle replication of configurations with default values.
+     * The configurations with default values are controlled by {@link #shouldReplicateSourceDefault(String)}.
```

Comment:
This part should be updated now. Maybe something like this?

```suggestion
     * Note that if a property has a default value on the source cluster,
     * {@link #shouldReplicateSourceDefault(String)} will also be called to
     * determine how that property should be synced.
```